src/services/staking_vaults.py

```diff
@@ -455,6 +456,12 @@ def calc_liquidity_fee(
                     minted_steth_on_event, blocks_elapsed_between_events, core_apr_ratio, liquidity_fee
                 )
 
+                if isinstance(event, VaultConnectedEvent):
```

Comment:
we should have one more event: `VaultConnectionUpdated` (as it changes the fee params too)

Reply:
cc: @tamtamchik (shall have the prev values)

Reply:
:brain: @tamtamchik can unite both methods there and have less code